src/include/migraphx/pass.hpp

```diff
@@ -333,6 +333,21 @@ inline const ValueType& any_cast(const pass& x)
 
 #endif
 
+/// Dummy pass for default return
+struct id_pass
+{
+    std::string name() const { return "id"; }
+    void apple(const module&) const {}
+};
+
+/// Used in the targets to enable/disable compiler passes
+inline pass enable_pass(bool enabled, pass p)
```

Comment:
This should be defined in a pass.cpp fle. And the `id_pass` class should be moved to the .cpp file as well.